pkg/alertmanager/multitenant.go

```diff
@@ -250,29 +315,150 @@ func createMultitenantAlertmanager(cfg *MultitenantAlertmanagerConfig, fallbackC
 		peer:                peer,
 		store:               store,
 		logger:              log.With(logger, "component", "MultiTenantAlertmanager"),
+		registry:            registerer,
+		ringCheckErrors: promauto.With(registerer).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_alertmanager_ring_check_errors_total",
```

Comment:
You need to document all introduced metrics in the `CHANGELOG`.

Reply:
I documented the ones I have introduced manually. However, the one introduces by the ring I have not. Should I?